File: app/src/main/java/com/vkpapps/soundbooster/interfaces/OnClientConnectionStateListener.java
package com.vkpapps.soundbooster.interfaces;

import com.vkpapps.soundbooster.connection.ClientHelper;

/*
 * @author VIJAY PATIDAR
 * */
public interface OnClientConnectionStateListener {
    void onClientConnected(ClientHelper clientHelper);

    void onClientDisconnected(ClientHelper clientHelper);
}

File: app/src/main/java/com/vkpapps/soundbooster/interfaces/OnClientConnectionStateListener.kt
package com.vkpapps.soundbooster.interfaces

import com.vkpapps.soundbooster.connection.ClientHelper

/***
 * @author VIJAY PATIDAR
 */


interface OnClientConnectionStateListener {
    fun onClientConnected(clientHelper: ClientHelper)
    fun onClientDisconnected(clientHelper: ClientHelper)
}